test_util/cluster_api.py

```diff
@@ -87,6 +87,24 @@ def _wait_for_dcos_history_up(self):
     @retrying.retry(wait_fixed=1000,
                     retry_on_result=lambda ret: ret is False,
                     retry_on_exception=lambda x: False)
+    def _wait_for_dcos_history_data(self):
+        for _ in range(10):
+            # call 10 times to make sure you get something, might need to sleep instead
```

Comment:
What is the purpose of iterating this ten times? If the request response is bad just return False to trigger the retry